### ISSUE TITLE
Any plan to update network trained from human games?

### ISSUE THREAD

**Issue by wonsiks** [leela-zero/leela-zero#1145]
Yes, I prefer a more human style so using weights.txt from best_v1.txt.zip which was made 5 months age.
I wonder if you have any plan to make further human style network?
If so, let me know when will it be please.

Or maybe I can get some from another way?

**Comment by ThorAvaTahr** [comment]
Human style has changed much over the last 5 months, I suppose. Perhaps you can simply use leela (original), AQ or another engine.

I don't see how this project (that intends to train a strong AI from zero human go-knowledge) can help you with your request. Especially as the weights file best_v1.txt.zip is already trained from 1.5 mio human games, there is not much more to do there. You could conceivably train a larger network from human games, but it would make more sense to switch to AQ for that.

**Comment by wonsiks** [comment]
AQ doesn’t match with lizzie. I want to train myself using leela-zero and lizzie.
But it’s okay even though stronger human style network doesn’t come out.
Just wonder about that.

**Comment by gcp** [comment]
It should be possible as better, larger human datasets have since come out. As always, it just requires someone to actually go ahead and do it rather than talk about it.

**Comment by ThorAvaTahr** [comment]
There would be little use, there are plenty of programs out there that are trained on human data.

**Comment by ThorAvaTahr** [comment]
I wonder if the OP understands that the _human style_ network is simply a network trained on human data. Such network consequently has a much lower overall strength than our current best network as it did not benefit from our shared effort to train a network with the self-play games.

**Comment by hydrogenpi** [comment]
@wonsiks its trivial to train a human network as long as you have access to enough games. anyone can do it on just a weak machine these days.

**Comment by wonsiks** [comment]
Yes, it's certain that human style network must be weaker than best network. There's no doubt.
However I'm gonna use leela-zero for educational purpose.
With some utility such as lizzie it's very useful for me to use leela-zero as a go teacher as long as I'm a beginner level player.

I think current human style network is good enough for that anyway.

Thanks for all~

**Comment by nauzykaa** [comment]
Look here: https://github.com/gcp/leela-zero/issues/884.

**Comment by wonsiks** [comment]
@nauzykaa Yeah, I have used it(#884) before but I felt that the original best_v1.txt.zip is better for me.
I can't explain why but just felt like that. And it was much heavy.
Thanks anyway.

**Comment by alreadydone** [comment]
@pangafu (and possibly others) are training Leela Master (128x20) and just produced their 7th network. It's trained using LZ self-play data and human/AI games online/offline and said to play much more like human, while also being stronger than official 128x10 LZ network.

**Comment by liverex** [comment]
As agreed by @pangafu, I uploaded the Leela Master weights V7 here:
https://drive.google.com/file/d/1MD_oAE9Ts03rKX3wNX4aK5kEpNNKY4zC/view?usp=sharing

**Comment by liverex** [comment]
If you need more training details, please ask @pangafu. He is currently on a trip.
@gcp, is it possible to schedule a match for it?

**Comment by gcp** [comment]
Note that this is much weaker than the "real" self-play networks.

**Comment by pangafu** [comment]
@gcp the upper leelamaster 20*128 network is not full-turning. I now training a 10*128 network with human sgf start from last 10b network 39d46507, and it will complete in few days, if I complete, I will post here, could you please schedule it match with 39d46507?

In my test, if a zero network train with leela zero sgf + human sgf, it almost have same strength , but the style will complete change to human, just like AI style transfer, if zero network train out a perfect CNN part, the style is just DNN part, and maybe the network strength is determined by CNN part.

**Comment by pangafu** [comment]
@gcp 
Here is the first human 10 * 128 network training from last 39d zero network, in my test, it may has the same strength with 39d46507 or better, when you play with it, it move more like human does. Could you schedule it match with 39d46507?

https://drive.google.com/drive/folders/1JOkKaw5VsojlbiTUGTm5QdqagavMWTRi?usp=sharing

![tensorboard](https://user-images.githubusercontent.com/10149168/38712458-6e96c040-3efe-11e8-89c6-917d23f1cd9f.png)

The last training score is(Learing rate 0.0001):
Accuracy:    Test 0.5734
MSE Loss:    Test 0.1638  /  Train 0.1629
Policy Loss: Test 1.421   /  Train 1.430

**Comment by pangafu** [comment]
And I will post all my trained leela master weight here

https://github.com/pangafu/LeelaMasterWeight

https://drive.google.com/drive/folders/1bB8ee1wFuRWL9nPhsl4_BPUhcWSBuxO0?usp=sharing
